cmd/git.go

```diff
@@ -10,6 +11,32 @@ import (
 
 type GitVCS struct{}
 
+var _ VCS = &GitVCS{}
+
+var (
+	WrongVCS            error = errors.New("Wrong VCS detected")
+	CannotDetermineRepo error = errors.New("Unable to determine repository")
+)
+
+var remoteRegex = regexp.MustCompile("^origin\\s+(\\S+)\\s+\\S+$")
+
+// returns the currently checked out remote repository
+// according to the state of the working directory
+func (g *GitVCS) currentRepository() (string, error) {
+	out, err := exec.Command("git", "remote", "-v").CombinedOutput()
```

Comment:
I checked out cookoo as an hg repo in the right place in the `_vendor` directory. Then I ran `glide update` from the root of the glide project. I saw it noted:
```
[INFO] Repository changed from git@github.com:Masterminds/glide.git to git@github.com:Masterminds/cookoo.git. I'm doing a clean checkout.
```
The wrong repo was listed as the one it's coming from. When `git remote -v` is run from the hg repo it bubbles up all the way to glide itself and provides that info rather than staying scoped to the _vendor directory.

For VCS changes this is wonkey. Any ideas how we could better handle this?